client/gameObjects/Skeleton.js

```diff
@@ -76,7 +93,25 @@ export default class Skeleton extends Phaser.GameObjects.Image {
     this.tick = 0;
     this.maxTick = getRandomInt(MIN_TICK, MAX_TICK);
 
-    this.hp = new HealthBar(scene, this.x, this.y, hp);
+    this.hp = new ValueBar(
```

Comment:
I would prefer to have it like
new HealthBar
new EnergyBar
I don't want to be aware of COLOR_RED and COLOR_ORANGE in the Skeleton constructor